app/forms/custom_form_helpers.rb

```diff
@@ -0,0 +1,24 @@
+module CustomFormHelpers
```

Comment:
Should this file not be called `phone_helpers.rb`.. otherwise the filename doesn't really say what it does.